src/lib/locais.json

```
@@ -122,4 +122,14 @@
             ]
         }
     ]
```

Falta uma virgula aqui.
```suggestion
    ],
```